src/main/java/org/jenkinsci/plugins/credentialsbinding/impl/ZipFileBinding.java

```diff
@@ -53,6 +54,7 @@
         secret.unzipFrom(credentials.getContent());
     }
 
+    @Symbol("zip")
```

Comment:
Pipeline utility steps has one - https://github.com/jenkinsci/pipeline-utility-steps-plugin/blob/master/docs/STEPS.md

In general, would suggest using slightly more descriptive name.